CHANGELOG.md

```diff
@@ -9,10 +9,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 - Added `Display`, `Error` and `PartialEq` implementations for `Error<E>`
+- Async support based on `embedded-hal-async` 1.0 behind `async` feature flag.
 
 ### Changed
 - [breaking-change] Removed `Default` implementation for `Pca9685` struct.
 - Raised MSRV to 1.62.0.
+- [breaking-change] Updated `embedded-hal-async` dependency to 1.0.
```

Comment:
```suggestion
```
Since the dependency was just added, there is no need to list this again.

Reply:
That's a typo from me, I meant to write `embedded-hal`